engine-standalone-storage/src/sync/types.rs

```diff
@@ -518,6 +521,7 @@ enum BorshableTransactionKind<'a> {
     Unknown,
     SetOwner(Cow<'a, parameters::SetOwnerArgs>),
     SubmitWithArgs(Cow<'a, parameters::SubmitArgs>),
+    SetUpgradeDelayBlocks(Cow<'a, parameters::SetUpgradeDelayBlocksArgs>),
```

Comment:
It should be after `FundXccSubAccound` because `FundXccSubAccound` is already [merged](https://github.com/aurora-is-near/aurora-engine/pull/735) and might be used already.